### PR TITLE
Add analytics settings commands

### PR DESCRIPTION
**Pull request by oNaiPs**

### CODE REVIEW

File: cmd/add-resources.go [hunk 1]
```diff
@@ -77,15 +75,7 @@ var resourcesAddCmd = &cobra.Command{
...
-						if s == "null" {
-							return
-						}
-						i, err := strconv.ParseUint(s, 10, 8)
-						if err != nil {
-							log.Fatal(err)
-						}
-						value := int64(i)
-						resource.FixedLastOctet.Value = &value
+						resource.FixedLastOctet.AssignFromString(s)
```
Comment (pablosole) [inline cmd/add-resources.go:78]: you put it on the other PR about settings

File: cmd/common-input.go [hunk 2]
```diff
@@ -482,6 +482,9 @@ func placeInputValues(cmd *cobra.Command,
 
 	switch entry.Type {
 	case individualValues:
+		if len(setterFuncs) != len(data.fields) {
```
Comment (pablosole) [inline cmd/common-input.go:485]: ditto

File: cmd/settings-analytics-set.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+		inputField{
+			Name:            "Disable SSL",
+			FlagName:        "external_server_disable_ssl",
+			FlagDescription: "Skips SSL server certificate checking for HTTPS events. WARNING: only use for development purposes.",
+			VarType:         "bool",
+			DefaultValue:    false,
+		},
+		inputField{
+			Name:            "Intercept All Domains",
+			FlagName:        "external_server_intercept_all_domains",
```
Comment (pablosole) [inline cmd/settings-analytics-set.go:117]: this flags are super verbose

**Review by pablosole — CHANGES_REQUESTED** [review]
I think you need to merge the other PR about settings first (already approved) and rebase this changes to it

**Comment by oNaiPs** [comment]
@pablosole fixed

File: cmd/settings-analytics-set.go (hunk 3 above)
Comment (oNaiPs) [reply cmd/settings-analytics-set.go]: fixed

**Review by pablosole — APPROVED** [review]

**Pull request merged**